sql/moz-fx-data-shared-prod/activity_stream/impression_stats_flat/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+friendly_name: Impressions stats flat
+workgroup_access:
+  - role: roles/bigquery.dataViewer
+    members:
+      - workgroup:pocket/external
```

Comment:
`workgroup_access` has been manually added for the downstream views `pocket.spoc_tile_ids` and `activity_stream_bi.impression_stats_by_experiment_v1`